src/rss.cpp

```diff
@@ -302,17 +302,19 @@ void rss_item::set_flags(const std::string& ff) {
 void rss_item::sort_flags() {
 	std::sort(flags_.begin(), flags_.end());
 
-	for (auto it=flags_.begin(); flags_.size() > 0 && it!=flags_.end(); ++it) {
+	auto it=flags_.begin(); 
```

Comment:
There's trailing whitespace on this line. Install [editorconfig](http://editorconfig.org/) plugin—it'll help with these details.